src/Storage/Storage/Common/Cmdlet/SetAzureStorageServiceMetrics.cs

```diff
@@ -172,11 +226,7 @@ public override void ExecuteCmdlet()
                 if ((MetricsType == ServiceMetricsType.Hour && currentServiceProperties.HourMetrics == null)
                     || (MetricsType == ServiceMetricsType.Minute && currentServiceProperties.MinuteMetrics == null))
                 {
-                    AccountProperties accountProperties = Channel.GetAccountProperties();
```

Comment:
These code should not be removed. 
Or it will always throw error no matter the account is premium or not.